packages/caliper-core/test/worker/tx-observers/prometheus-push-tx-observer.js

```diff
@@ -216,7 +216,8 @@ describe('When using a PrometheusPushTxObserver', () => {
 
         await prometheusPushTxObserver.deactivate();
 
-        // Values should be zero, or empty (https://github.com/siimon/prom-client/blob/master/test/counterTest.js)
+        // Values should be zero, or empty 
+        // Ref: https://github.com/siimon/prom-client/  file test/counterTest.js
```

Comment:
this url doesn't look valid. If we can't use the real url then we should delete the line

Reply:
It's split into repo and file parts. I could structure it better if needed. But the entire file URL contains `master`...
Or we can remove it entirely

Reply:
@davidkel, could be structured like this:
`See the "test/counterTest.js" file in https://github.com/siimon/prom-client/`